src/app/docs/page/page.component.ts

```diff
@@ -76,6 +76,17 @@ export class PageComponent implements OnInit, AfterViewInit {
           }
         }, 200);
       });
+      if (this.slug === 'blog') {
+        const doc = document;
```

Comment:
These 9 lines look near identical to the home.component.ts.  See if you can move it to a service or something.